### DISCUSSION TITLE
issue: opensearch.xml // "+ Add LibreY search" to the right click on the address bar.

### DISCUSSION THREAD

**Discussion by rjc3rd** [Ahwxorg/LibreY#161] [General]
I hard forked my instance of LibreY search and made deep cuts into the folder structure at first. My instance does not offer the option to Add LibreWTF search to my browser. 

I made a proper fork of LibreY on the same server and it does offer the Add LibreY search on right click in the address bar.

I assume I accidentally removed something I needed. Does anyone know where I would find that?

**Comment by Ahwxorg** [comment]
I think you need opensearch.xml.

**Reply by Ahwxorg** [reply]
(at the root of the domain, such as search.ahwx.org/opensearch.xml

**Reply by rjc3rd** [reply]
Thank you. You sent me in the right direction.

I did not update opensearch.xml file after I had dropped the libre.wtf domain and just started using the duckdns.org domain. Plus I found a small issue with my misc/header.php file that may caused the line linking to the opensearch.xml to not being able to load the file anyway. 

I got it working now.

**Reply by Ahwxorg** [reply]
Nice!
